tests/garage/tf/algos/test_td3.py

```diff
@@ -0,0 +1,74 @@
+"""Create a test that fails when garage.tf.algos.TD3 performance is too low."""
+
+import gym
+import tensorflow as tf
+
+from garage.np.exploration_strategies.gaussian_strategy import GaussianStrategy
+from garage.replay_buffer import SimpleReplayBuffer
+from garage.tf.algos import TD3
+from garage.tf.envs import TfEnv
+from garage.tf.experiment import LocalTFRunner
+from garage.tf.policies import ContinuousMLPPolicy
+from garage.tf.q_functions import ContinuousMLPQFunction
+from tests.fixtures import snapshot_config, TfGraphTestCase
+
+
+class TestTD3(TfGraphTestCase):
+    """Tests for TD3 algo."""
+
+    def test_td3_pendulum(self):
+        """Test TD3 with Pendulum environment."""
+        with LocalTFRunner(snapshot_config) as runner:
+            env = TfEnv(gym.make('InvertedDoublePendulum-v2'))
+
+            action_noise = GaussianStrategy(env.spec,
+                                            max_sigma=0.1,
+                                            min_sigma=0.1)
+
+            policy = ContinuousMLPPolicy(env_spec=env.spec,
+                                         hidden_sizes=[400, 300],
+                                         hidden_nonlinearity=tf.nn.relu,
+                                         output_nonlinearity=tf.nn.tanh)
+
+            qf = ContinuousMLPQFunction(name='ContinuousMLPQFunction',
+                                        env_spec=env.spec,
+                                        hidden_sizes=[400, 300],
+                                        action_merge_layer=0,
+                                        hidden_nonlinearity=tf.nn.relu)
+
+            qf2 = ContinuousMLPQFunction(name='ContinuousMLPQFunction2',
+                                         env_spec=env.spec,
+                                         hidden_sizes=[400, 300],
+                                         action_merge_layer=0,
+                                         hidden_nonlinearity=tf.nn.relu)
+
+            replay_buffer = SimpleReplayBuffer(env_spec=env.spec,
+                                               size_in_transitions=int(1e6),
+                                               time_horizon=250)
+
+            algo = TD3(env_spec=env.spec,
+                       policy=policy,
+                       policy_lr=1e-3,
+                       qf_lr=1e-3,
+                       qf=qf,
+                       qf2=qf2,
+                       replay_buffer=replay_buffer,
+                       target_update_tau=0.005,
+                       n_epoch_cycles=20,
```

Comment:
aren't these two params handled by the runner?

@naeioi @zequnyu can you take a look to make sure this is using the runner API properly?

Reply:
For now I think it is. I actually had some problems about #736 and want to clarify with @naeioi. Maybe I should do a separate PR to remove `n_epoch_cycles` from algos first.

Reply:
Zequn and I just discussed on this. `n_epoch_cycles` is duplicate on algorithm and runner's interface but it is required for training. We will have a PR to fix this later, but let's not put it in this PR.